common-api/src/main/java/org/pmiops/workbench/cdr/dao/CBCriteriaDao.java

```diff
@@ -83,16 +83,16 @@
   /** This query returns all parents matching the parentConceptIds. */
   @Query(
       value =
-          "select c from CBCriteria c where conceptId in (:parentConceptIds) and domainId = :domain and type = :type and standard = :standard and "
+          "select c from DbCriteria c where conceptId in (:parentConceptIds) and domainId = :domain and type = :type and standard = :standard and "
```

Comment:
Double check that this is what's wanted.

Reply:
Yes, intended.
You probably already know this but will toss this out there just in case. 
This `@Query` annotation is using JPQL, so the it uses the object name instead of the table name. 
JPQL Example:
`@Query("select obj from DbMyObject obj where obj.status = 1")`
Native Example:
`@Query("select * from my_table where status = 1", nativeQuery = true)`

Reply:
Cool. I wish they'd made the native one a separate annotation instead of an option on this one, but it works.